File: 코딩 기초 트레이닝/배열의 길이에 따라 다른 계산하기-p181854/배열의 길이에 따라 다른 계산하기.js
function solution(arr, n) {
    var answer = [];

    arr.forEach((d, index) => {
        if(arr.length % 2 === 0) {
            // arr 길이 짝수
            answer.push(d + (index % 2 === 0 ? 0 : n));
        } else {
            // arr 길이 홀수
            answer.push(d + (index % 2 !== 0 ? 0 : n));
        }
    });

    return answer;
}

File: 코딩 기초 트레이닝/배열의 길이에 따라 다른 계산하기-p181854/배열의 길이에 따라 다른 계산하기.py
def solution(arr, n):
    answer = []

    # flag == 0 짝수, != 0 홀수
    flag = len(arr) % 2


    if flag : #홀수
        for index, d in enumerate(arr):
            answer.append(d + (0 if index % 2 != 0 else n))
    else : # 짝수
        for index, d in enumerate(arr):
            answer.append(d + (0 if index % 2 == 0 else n))

    return answer
